azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+jobs:
+- job: test
+  displayName: 'Lint & Test'
+
+  pool:
+    vmImage: 'Ubuntu 16.04'
+
+  steps:
+  - script: sudo apt-get install docker
+    displayName: 'Install base dependencies'
+
+  - script: docker build -t cd-bot .
+    displayName: 'Build docker container'
+
+  - script: pip install -r requirements-ci.txt
+    displayName: 'Install project using pipenv'
+
+  - script: python -m flake8 .
+    displayName: 'Run linter'
```

Comment:
Codacy is dead, long live Codacy

Reply:
is this a review comment or a reflection on codacy

Reply:
![tenor](https://user-images.githubusercontent.com/3729048/54078701-e8e73300-42c4-11e9-8068-4c4348e92566.gif)